protos/action/action.proto

```diff
@@ -166,6 +170,14 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+message SetActuatorRequest {
+  int32 index = 1; // Index of actuator (starting with 1)
+  float value = 2; // Value to set the actuator to (normalized from [-1..1])
```

Comment:
Ha I found something to complain! :smile: 

Please match indentation to 4 spaces everywhere.

Reply:
Whoops, not sure how that happened. Will fix